Formula/metview.rb

```diff
@@ -0,0 +1,87 @@
+class Metview < Formula
+  desc "Meteorological workstation software"
+  homepage "https://metview.readthedocs.io/en/latest/"
+  url "https://confluence.ecmwf.int/download/attachments/51731119/MetviewBundle-2022.8.1-Source.tar.gz"
+  version "5.17.1"
+  sha256 "f9eb91918e2bc43388d53cb424de95fa352a3cb64f01cbd0dbd62303a05086f9"
+  license "Apache-2.0"
+
+  # NOTE: do not build on linux because of libtirpc problems - come back to it later
```

Comment:
What problems?

Reply:
Our CMake detection of tirpc can require some tweaking, especially on Ubuntu, where the libs are not installed into /usr/lib by the normal package manager. I guess it might be simpler with brew, but I'd prefer to try it with a test system locally rather than try/fail/repeat with GitHub pushes. And macOS is our target system at the moment, so we'd like to get it available there before adding Linux support.

Reply:
Unless the software is not meant to run on Linux I'd prefer not to ship half support in Homebrew.

Reply:
OK, let me try to get that working.

Reply:
Done, now working on Linux.